content/workshops/rhel_8/exercise1.0.adoc

```diff
@@ -0,0 +1,236 @@
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 12: Kernel Dump Module
+
+image::cockpit_11_kd.png[caption="Figure 13: ", title='Web Console Kernel Dump Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 13: SELinux Module
+
+image::cockpit_12_selinux.png[caption="Figure 14: ", title='Web Console SELinux Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 14: Software Update Module
+
+image::cockpit_13_su.png[caption="Figure 15: ", title='Web Console Kernel Dump Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 15: Subscriptions Module
+
+image::cockpit_14_subscriptions.png[caption="Figure 16: ", title='Web Console Subscriptions Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 16: Terminal Module
+
+image::cockpit_15_terminal.png[caption="Figure 17: ", title='Web Console Terminal Module', 500]
+
+The terminal module provides command-line access to the system, similar to an SSH session.  Section 16.1 has some example commands to try if you'd like to test it out.
+
+==== Section 16.1: Example terminal usage
+
+===== Example 1: Use the following command to pull a RHEL image from the Red Hat repository:
+
+[source, bash]
+----
+podman pull registry.access.redhat.com/ubi8/ubi-minimal
+----
+
+Now if you return to the Podman Containers tab (above, in Section 5), you should see this image listed.  You'll see more of podman later in Exercise 4.
+
+===== Example 2: Try the Skopeo command
+
+Skopeo provides the ability to inspect container images stored in a registry. The Inspect feature displays details of the selected image. The 'skopeo inspect' command's ability to list all the tags associated with a selected image is a benefit over the docker tool.
+
+[source, bash]
+----
+skopeo inspect docker://registry.access.redhat.com/ubi8/ubi-minimal
+----
+
+===== Example 3: List firewall rules
+
+nft is the nftables utility.  Similar to ''iptables -L'', the following lists the system's active firewall ruleset:
+
+[source, bash]
+----
+nft list ruleset
+----
+
+=== Conclusion
+
+The web console has a lot of new functionality, in version 8 of Red Hat Enterprise Linux. Hopefully, this brief (!) overview has demonstrated that, and given you a taste of how easily systems may be managed, with this new tool.
```

Comment:
Line 234 The web console in Red Hat Enterprise Linux Version 8 has a lot of new functionality. Hopefully, this brief (!) overview has provided a comprehensive description of the web console features and provided you with a taste of how you can easily manage your systems with this new tool.